Tensile/TensileCreateLibrary.py

```diff
@@ -1280,6 +1282,12 @@ def TensileCreateLibrary():
                           default=False, help="Output manifest file with list of expected library objects and exit.")
   argParser.add_argument("--library-format", dest="LibraryFormat", choices=["yaml", "msgpack"], \
       action="store", default="msgpack", help="select which library format to use")
+  argParser.add_argument("--generate-sources-and-exit",   dest="GenerateSourcesAndExit", action="store_true",
+                          default=False, help="Output source files only and exit.")
+  argParser.add_argument("-j",   dest="CpuThreads", type=int,
+                          default=-1, help="Number of parallel jobs to launch.")
+  argParser.add_argument("-v",   dest="PrintLevel", type=int,
+                          default=-1, help="Set printout verbosity level.")
```

Comment:
Good idea to make this more canonical.

Reply:
Do you mean to make them `--jobs` and `--verbose`? Or do you prefer them to match `CpuThreads`/`PrintLevel`?

Reply:
Updated to accept both `--jobs|-j` and `--verbose|-v`.